content/chronograf/v1.9/about_the_project/release-notes-changelog.md

```diff
@@ -8,6 +8,43 @@ menu:
     parent: About the project
 ---
 
+## v1.9.4 [2022-03-28]
+
+### Features
+
+This release renames the Flux `Query Builder` to the Flux `Script Builder` (and adds improvements), and improves on Kapacitor integration.
+
+#### Flux Builder improvements
+
+- Rename the Flux `Query Builder` to the Flux `Script Builder`, and add new functionality including:
+  - Ability to load truncated data into the Flux Script Builder.
+  - Script Builder `tag keys` and `tag values` depend on a selected time range.
+- Choose a time range for data output in Flux schema explorer, reducing query time.
```

Comment:
remove.  This is the same fix as mentioned above.  The schema range is driven by the time selector....for both Script Editor and Script Builder.

Reply:
removed line 22